Global/JetBrains.gitignore

```diff
@@ -21,6 +21,14 @@
 .idea/**/gradle.xml
 .idea/**/libraries
 
+# Gradle and Maven with auto-import
+# When using Gradle or Maven with auto-import, you should exclude module files,
+# since they will be recreated, and may cause churn.  Uncomment if using
+# auto-import.
+# .idea/modules.xml
+# .idea/**/*.iml
```

Comment:
Can this be more specialized than all `.iml` files? This seems to clash with an earlier rule:

> All the `.iml` module files that can be located in different module directories (applies to IntelliJ IDEA)

Reply:
in the .idea directory, there is the project .iml (.idea/projectname.iml) and the module .iml's (.idea/modules/**/*.iml).  I am still not clear why these iml files sometimes end up in .idea/modules, and sometimes in the individual module directories, but in either case, that is what the author considers in the first bullet point under that to consider not to share.

If you would like, I can make it more explicit by breaking it up into:
.idea/*.iml
.idea/modules/**/*.iml